### DISCUSSION TITLE
If i have tracks moved with picard (not renamed) from one folder to another, is there a way to keep track ratings?

### DISCUSSION THREAD

**Discussion by jsrol** [navidrome/navidrome#2316] [Q&A] [answered]
Question in title. I have tried it and it doesnt seem to keep the ratings, i just want to make sure this is how it's supposed to be and i didnt make a mistake. Maybe there's some clever way to do this?

Basically my usecase is i am currently tagging my collection with Picard but would like to have the tracks moved to a "tagged" folder when completed so that i can see where i'm at in the tagging job. I have a few tens of thousands of tracks with a few thousands that are already rated. I would like to be able to keep these ratings while i slowly tag my albums.

Folder structure is /Music/Artist/Album. New location is /Music/Tagged/Artist/Album. 

Tracks are NOT tagged in one go. It will take me months/years to finish this. Is there a way to keep track rating in an efficient way?

**Comment by deluan** [comment, marked as answer]
Yeah, this is caused by the way the scanner works today. There are plans to change the way we identify songs/albums/artists, that will solve this in a future version. The issue to follow is this one: https://github.com/navidrome/navidrome/issues/1976#issuecomment-1383278483

**Reply by jsrol** [reply]
Thanks! I'll refrain to move files until then.
